src/components/authentication/authentication.service.ts

```diff
@@ -419,9 +420,11 @@ export class AuthenticationService {
     }
   }
 
-  private get argon2Options() {
-    return this.config.passwordSecret
-      ? { secret: Buffer.from(this.config.passwordSecret, 'utf-8') }
-      : {};
+  private get argon2Options(): Except<argon2.Options, 'raw'> {
+    return {
+      secret: this.config.passwordSecret
+        ? Buffer.from(this.config.passwordSecret, 'utf-8')
+        : undefined,
```

Comment:
I don't think `undefined` works for argon2.Options

Reply:
Really? Ok good catch.